include/zephyr/net/wifi.h

```diff
@@ -87,6 +89,12 @@ enum wifi_security_type {
 	/** FT-EAP-SHA384 security */
 	WIFI_SECURITY_TYPE_FT_EAP_SHA384,
 
+	/** WPA/WPA2/WPA3 PSK security, sae with hash-to-element. */
+	WIFI_SECURITY_TYPE_WPA_AUTO_PERSONAL_H2E,
+	/** WPA/WPA2/WPA3 PSK security, sae with both hunting-and-pecking
+	 *  loop and hash-to-element enabled.
+	 */
+	WIFI_SECURITY_TYPE_WPA_AUTO_PERSONAL_AUTO,
```

Comment:
This is a bit confusing, seeing two AUTOs :). May be drop this and reuse `WIFI_SECURITY_TYPE_WPA_AUTO_PERSONAL` to mean everything is auto?

Reply:
Yes, of course, it should be like this, actually I notice this point early.
updated as comment.